src/deploy/functions/runtimes/discovery/parsing.ts

```diff
@@ -106,6 +107,14 @@ export function assertKeyTypes<T extends object>(
       }
       continue;
     }
+    if (schemaType === "List") {
```

Comment:
Technically you've defined "List?" as a type. Do you want to check for value === null first or is that case already handled?

Reply:
It looks to me like the conditional checks immediately above that one cover this case, unless I'm badly misinterpreting your comment.